mkdocs.yml

```diff
@@ -25,6 +25,7 @@ pages:
   - 'Log4j2 Appender': ext/log4j2.md
   - 'Log4j1 Appender': ext/log4j1.md
   - 'Placeholders': ext/placeholders.md
+  - 'Thread Pools': ext/thread-pools.md
```

Comment:
For the extensions, I think we should either sort the list alphabetically or group this with other JVM stats. For example, maybe put it under "Memory Pools".

Reply:
Probably makes sense to put it with other JVM stats, but also introduce that as a parent.  Without the additional level, it makes more sense to have a flat alphabetized list.